src/kem/kyber/kem_kyber_1024.c

```diff
@@ -30,64 +30,64 @@ OQS_KEM *OQS_KEM_kyber_1024_new() {
 	return kem;
 }
 
-extern int PQCLEAN_KYBER1024_CLEAN_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
-extern int PQCLEAN_KYBER1024_CLEAN_crypto_kem_enc(unsigned char *ct, unsigned char *ss, const unsigned char *pk);
-extern int PQCLEAN_KYBER1024_CLEAN_crypto_kem_dec(unsigned char *ss, const unsigned char *ct, const unsigned char *sk);
+extern int pqcrystals_kyber1024_ref_keypair(unsigned char *pk, unsigned char *sk);
+extern int pqcrystals_kyber1024_ref_enc(unsigned char *ct, unsigned char *ss, const unsigned char *pk);
+extern int pqcrystals_kyber1024_ref_dec(unsigned char *ss, const unsigned char *ct, const unsigned char *sk);
 
 #if defined(OQS_ENABLE_KEM_kyber_1024_avx2)
-extern int PQCLEAN_KYBER1024_AVX2_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
-extern int PQCLEAN_KYBER1024_AVX2_crypto_kem_enc(unsigned char *ct, unsigned char *ss, const unsigned char *pk);
-extern int PQCLEAN_KYBER1024_AVX2_crypto_kem_dec(unsigned char *ss, const unsigned char *ct, const unsigned char *sk);
+extern int pqcrystals_kyber1024_avx2_keypair(unsigned char *pk, unsigned char *sk);
+extern int pqcrystals_kyber1024_avx2_enc(unsigned char *ct, unsigned char *ss, const unsigned char *pk);
+extern int pqcrystals_kyber1024_avx2_dec(unsigned char *ss, const unsigned char *ct, const unsigned char *sk);
 #endif
 
 OQS_API OQS_STATUS OQS_KEM_kyber_1024_keypair(uint8_t *public_key, uint8_t *secret_key) {
 #if defined(OQS_ENABLE_KEM_kyber_1024_avx2)
 #if defined(OQS_PORTABLE_BUILD)
 	OQS_CPU_EXTENSIONS available_cpu_extensions = OQS_get_available_CPU_extensions();
-	if (available_cpu_extensions.AES_ENABLED && available_cpu_extensions.AVX2_ENABLED && available_cpu_extensions.BMI2_ENABLED && available_cpu_extensions.POPCNT_ENABLED) {
+	if (available_cpu_extensions.AVX2_ENABLED && available_cpu_extensions.BMI2_ENABLED && available_cpu_extensions.POPCNT_ENABLED) {
```

Comment:
This (generated) code seems to contradict the documentation: "**Optimizations**: Portable C with AES, AVX2, POPCNT, SSE2 and SSSE3 instructions (if available at run-time)". Question: Did you build-and-run this branch locally once with OQS_USE_CPU_EXTENSIONS=ON and once with OFF and see the expected performance differences?

Reply:
Thanks for the suggestion for testing with OQS_USE_CPU_EXTENSIONS=ON/OFF. The performance difference is there as expected.